### ISSUE TITLE
BitBucket support

### ISSUE THREAD

**Issue by pogo19** [netlify/git-gateway#13]
Please add BitBucket support, thanks!

**Comment by Wandern** [comment]
Very interested in this as well. Github is all well and good, unless your project requires the use of private repositories, with minimal cost.

**Comment by cNille** [comment]
Is there any progress with this issue? 
I totally agree with Wandern about this.

**Comment by lauragift21** [comment]
It will be very nice to have BitBucket support too I'm in a situation where I need this for a project.

**Comment by ajmueller** [comment]
GitHub has had support for free Private repositories for a couple years now, so for those who held back on using GitHub for this reason, you can give it a try now 👍

**Comment by timtorres** [comment]
Echoing this request - I don't have the option to use GitHub for a project at work.

**Comment by erezrokah** [comment]
> And I know this isn't the place for Netlify CMS but it's also weird this isn't explicitly called out in the docs there.

Hi @timtorres, for the CMS docs see https://www.netlifycms.org/docs/git-gateway-backend/.
Should we link to it from another part of the docs to make it more clear? Can you open an issue for it on the CMS repo of where you would expect it to be stated?

**Comment by timtorres** [comment]
@erezrokah Yes I went through that and the [Bitbucket](https://www.netlifycms.org/docs/bitbucket-backend/) page several times trying to understand.
It's more about the wording than the location, maybe a mention somewhere else could help. It does give alternate instructions for Bitbucket but doesn't make it clear that it's ⚠️ not compatible with Netlify Identity ⚠️, nor make clear the implications, etc. If we have more than 5 users, we'd now have to pay for a Bitbucket plan in addition to Netlify, and they'd all need repo access where before I could have unlimited editors interact with Netlify Identity only. These details and no clear roadmap for integration in the last few years would've had a direct impact on the stack I chose. I know there are alternatives but they're all a bit advanced for me.

But yes, sorry, this is just for context – I'll see if I can open an issue with some helpful suggestions.
